packages/lesswrong/lib/instanceSettings.ts

```diff
@@ -138,6 +138,29 @@ export const taggingNameCapitalSetting = {get: () => startCase(taggingNameSettin
 export const taggingNamePluralSetting = {get: () => pluralize(taggingNameSetting.get())}
 export const taggingNamePluralCapitalSetting = {get: () => pluralize(startCase(taggingNameSetting.get()))}
 export const taggingNameIsSet = {get: () => taggingNameSetting.get() !== 'tag'}
+export const taggingNameIsPluralized = {get: () => !isLWorAF && taggingNameIsSet.get()};
```

Comment:
FYI: this block is probably one of the highest-risk changes from the EA forum's perspective and worth testing thoroughly

Reply:
It did indeed break one thing: https://github.com/ForumMagnum/ForumMagnum/pull/10251#issuecomment-2631392929